taxi/views.py

```diff
@@ -85,6 +138,29 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
```

Comment:
add additional typehint for list inner objects like list[ANY_TYPE]
use typehint like: Optional[YOUR_VALUE_TYPE] = None